package.json

```diff
@@ -79,6 +79,7 @@
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-mocha": "^5.2.0",
```

Comment:
Why do we need this?

Reply:
Honestly not sure, I was having trouble with linting. This error message led me to get that and after I did, it worked.
<img width="707" alt="error" src="https://user-images.githubusercontent.com/29712119/49565078-c9fa8e80-f8da-11e8-8972-7617c42d1c5d.png">